src/Sets/HalfSpace.jl

```diff
@@ -527,6 +527,44 @@ _normal_Vector(c::LinearConstraint) = LinearConstraint(convert(Vector, c.a), c.b
 function load_modeling_toolkit_halfspace()
 return quote
 
+# returns `(true, sexpr)` if expr represents a half-space,
+# where sexpr is the simpified expression sexpr := LHS - RHS <= 0
+# otherwise, returns `(false, expr)`
```

Comment:
Should it rather return a dummy `sexpr` for type stability? Since this code is not performance critical, it is fine if not.

Reply:
Both `expr` and `sexpr` are of the same type `Operation` (no parameters):

```julia
julia> expr = x + y ≥ 1
(1 < x + y) | (1 == x + y)

julia> valid, sexpr = LazySets._is_halfspace(expr)
(true, 1 + -1 * (x + y))

julia> typeof(expr)
Operation

julia> typeof(sexpr)
Operation

julia> expr = x + y == 2
x + y == 2

julia> valid, sexpr = LazySets._is_hyperplane(expr)
(true, -2 + x + y)

julia> typeof(sexpr)
Operation
```

Reply:
Ah, never mind then :grinning: